system/tarslite/templates/ingress.yaml

```diff
@@ -19,5 +20,29 @@ spec:
                 port:
                   name: webhook
   tls:
-    - hosts: [ {{ $hostname }} ]
+    - hosts: [{{ $hostname }}]
       secretName: tls-{{ .Release.Name }}
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: alertmanager-rule
+  annotations:
```

Comment:
Please add the annotations to configure Basic Auth for this endpoint

Reply:
Annotation added for basic auth